plugin.xml

```diff
@@ -113,6 +113,6 @@ xmlns:android="http://schemas.android.com/apk/res/android">
 	</platform>
 
 	<hook src="scripts/after_prepare.js" type="after_prepare" />
-	<hook src="scripts/after_plugin_add.js" type="after_plugin_add" />
+	<hook src="scripts/after_plugin_install.js" type="after_plugin_install" />
 	<hook src="scripts/before_plugin_rm.js" type="before_plugin_rm" />
```

Comment:
It seems that we should be consistent in the install and uninstall events.  If we modify on install, we should modify on uninstall instead of remove.  I could not find the difference between the two events, but since we are changing add > install, we should change rm > uninstall to be consistent.